_versions/babelfish-1.4.0.md

```diff
@@ -0,0 +1,25 @@
+---
+date: 2022-7-25
```

Comment:
Please fix the date.

Reply:
I'll update that after we upload the installers/release notes - it's easier/safer that way.